release/artifacts.yaml

```diff
@@ -1,25 +1,36 @@
-- target: //daml-lf/archive:daml_lf_archive_java
-  type: jar
-  javadoc-jar: daml_lf_archive_java_javadoc.jar
-  src-jar: daml_lf_archive_java_srcjar.jar
-  mavenUpload: True
 - target: //daml-lf/data:data
   type: jar-scala
-- target: //compiler/damlc/jar:damlc_jar
-  type: jar-deploy
-  platformDependent: True
-- target: //daml-lf/archive:daml_lf_archive_scala
-  type: jar-scala
-- target: //daml-lf/archive:daml_lf_archive_protos_zip
+- target: //daml-lf/archive:daml_lf_archive_java_proto
```

Comment:
Since this represents the `1.dev` version, `dev` should also be part of the bazel target name and the maven coordinates.

Reply:
done.